### DISCUSSION TITLE
upgraded now admin gets fatal error

### DISCUSSION THREAD

**Discussion by raggoo** [YOURLS/YOURLS#3094] [General]
Upgraded from v172 to v182 and now try to login to mysite/admin and get:
Fatal error
Incorrect DB config, or could not connect to DB
PDOException: SQLSTATE[HY000] [2054] The server requested authentication method unknown to the client

What’s wrong please and how to fix?

Have I lost my database? (years old)

**Comment by ozh** [comment]
See https://github.com/YOURLS/YOURLS/wiki/Troubleshooting-first-steps
